packages/apps/plugins/plugin-native/src/NativePlugin.tsx

```diff
@@ -48,19 +50,24 @@ const initializeNativeApp = async (plugins: Plugin[]) => {
   //
 
   let itemsMac = '';
+  // TODO(mjamesderocher) This isn't working
   if (process.platform === 'darwin') {
     itemsMac = `
-      Hide: h + CommandOrControl
+      Hide ${appName}: h + CommandOrControl
       Hide Others: h + Control + Meta
       ---
     `;
   }
 
+  // TODO(mjamesderocher) Change menu names to use translations
```

Comment:
@wittjosiah Is there a way to get translations from other plugins? I would specifically need the translations from plugin-navtree.

This might be something we will just handle later. I'll let you make the judgment call on it.

Reply:
Currently the translations utils are only available in react-land unfortunately. The plan is to expose them at the plugin level as well for cases like this (at which point you'd be able to just use the plugin id as the translation namespace to access them), for now it's fine to hardcode this.